src/qililab/pulse/pulse_shape/drag.py

```diff
@@ -50,13 +50,15 @@ class Drag(PulseShape):
         drag_coefficient (float): Drag coefficient that give the DRAG its imaginary components.
     """
 
-    name = PulseShapeName.DRAG
-    num_sigmas: float
-    drag_coefficient: float
+    name = PulseShapeName.DRAG  #: Name of the drag pulse shape.
+    num_sigmas: float  #: Sigma number of the drag pulse shape.
+    drag_coefficient: float  #: Drag coefficient.
```

Comment:
aren't these redundant since they are already defined in `Args`?

Reply:
agree, addressed.